README.md

```diff
@@ -86,7 +86,7 @@ pip install gtn
 First, clone the project:
 
 ```
-git clone git@github.com:facebookresearch/gtn.git && cd gtn
```

Comment:
Using "git@github.com" requires the user to setup his/her GitHub SSH key, which can be difficult for many users.

I would suggest using "https://GitHub.com"